src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -231,7 +231,8 @@ void should_return_a_200_and_task_list_when_idam_user_id_same_with_task_assignee
         List<Map<String, CamundaVariable>> mockedResponse = asList(Map.of(
             "taskType", new CamundaVariable("reviewTheAppeal", "String"),
             "completionMode", new CamundaVariable("Auto", "String"),
-            "workType", new CamundaVariable("decision_making_work", "String")
+            "workType", new CamundaVariable("decision_making_work", "String"),
+            "description", new CamundaVariable("aDescription", "String")
```

Comment:
this is R2 feature. It should be null

Reply:
You might be correct, since this is R1 test, i will remove the description from the mocked response